src/sympc/approximations/utils.py

```diff
@@ -0,0 +1,13 @@
+def sign(data):
```

Comment:
Q: I am not sure what is the difference between ```signum``` and ```sign```

Reply:
ahh it was redundant removed it we now have only sign